src/operation-node/data-type-node.ts

```diff
@@ -4,6 +4,7 @@ import { OperationNode } from './operation-node.js'
 export type ColumnDataType =
   | 'varchar'
   | `varchar(${number})`
+  | `char(${number})`
```

Comment:
What about just `char`?

Reply:
I assumed `char` without size didn't existed as its fixed size. I apologize for that, added it now, and read the docs for both mysql and postgres and found that `char` is equivalent to `char(1)`.

Thanks for pointing this out @igalklebanov :+1: